### PR TITLE
fix(suite): device label input

### PR DESCRIPTION
**Pull request by AdamSchinzel**
Device label input tweaks

## Description

Device label input has the same width as button and also fixed mobile version.

## Screenshots:

Screenshot 2023-08-23 at 12 26 51 PM
Screenshot 2023-08-23 at 12 27 00 PM

### CODE REVIEW

File: packages/components/src/components/form/Input/Input.tsx [hunk 1]
```diff
@@ -181,6 +187,7 @@ const Input = ({
     return (
         <Wrapper
             width={width}
+            alignInputRight={alignInputRight}
```
Comment (dahaca) [inline packages/components/src/components/form/Input/Input.tsx]: Please, do not introduce properties to global components which will only be used once. Create a SyledInput and apply alignment there if needed :)

File: packages/components/src/components/form/Input/Input.tsx [hunk 2]
```diff
@@ -16,10 +16,14 @@ import {
     LabelAddon,
 } from '../InputStyles';
 
-const Wrapper = styled.div<Pick<InputProps, 'width'>>`
+const Wrapper = styled.div<Pick<InputProps, 'width' | 'alignInputRight'>>`
     display: inline-flex;
     flex-direction: column;
     width: ${({ width }) => (width ? `${width}px` : '100%')};
+
+    @media (min-width: ${SCREEN_SIZE.SM}) {
```
Comment (dahaca) [inline packages/components/src/components/form/Input/Input.tsx]: Better use `SCREEN_QUERY`

**Review by dahaca — CHANGES_REQUESTED** [review]

File: packages/components/src/components/form/Input/Input.tsx (hunk 1 above)
Comment (AdamSchinzel) [reply packages/components/src/components/form/Input/Input.tsx]: 695eeef2c9bcbb469a40854c16adc4b9335722f4

File: packages/components/src/components/form/Input/Input.tsx (hunk 2 above)
Comment (AdamSchinzel) [reply packages/components/src/components/form/Input/Input.tsx]: 695eeef2c9bcbb469a40854c16adc4b9335722f4

**Review by dahaca — APPROVED** [review]

**Comment by bosomt** [comment]
**QA OK**


**Info:**
 - Suite version: desktop 23.9.1 (41a9c13bf28edf71fe629379288a5ad685f259d1)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/23.9.1 Chrome/112.0.5615.204 Electron/24.5.1 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: Trezor T2T1 2.6.0 regular (revision 88e1f8c7a5c7615723664c64b0a25adc0c409dee)
 - Transport: BridgeTransport 2.0.33

**Pull request merged**